tests/common/macros/comparators.ts

```diff
@@ -29,7 +29,7 @@ export function expectModifiedClonedValidator<T>(expected: BaseValidator<T>, act
 export function expectError<T = any>(cb: () => T, expected: BaseError) {
 	try {
 		cb();
-		fail('expected to throw, but failed to do so');
+		fail();
```

Comment:
This should be moved outside the try/catch block, and be called with `'Expected to throw, but failed to do so'`

Reply:
A `return;` at the end of the catch block allows us to not call `fail()` if the callback threw.